src/main/java/Event.java

```diff
@@ -0,0 +1,33 @@
+import java.time.format.DateTimeFormatter;
+import java.time.DateTimeException;
+import java.time.LocalDate;
+public class Event extends Task {
+
+    protected LocalDate from;
+    protected LocalDate to;
+
+    public Event (String description, String from, String to) {
+        super(description);
+        try {
+            this.from = LocalDate.parse(from);
+            this.to = LocalDate.parse(to);
+        } catch (DateTimeException e) {
+            throw new DateTimeException("Make sure the deadline in yyyy-mm-dd");
+        }
+    }
+
+    @Override
+    public String getTaskType() {
+        return "E";
+    }
+    
+    @Override 
+    public String getTimeline() {
+        return this.from.format(DateTimeFormatter.ofPattern("yyyy-MM-dd")) + "," + this.to.format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Maybe separating this one return statement into 3 lines of code would make it more readable?